### ISSUE TITLE
No status bar

### ISSUE THREAD

**Issue by LosTigeros** [wasdennnoch/AndroidN-ify#1263]
**Description**
When I want to pull down (expand) status bar, its shows nothing (there is only transparent background coming down).

**Module version (build version)**
jenkins-AndroidN-ify-788

**Xposed log / logcat**
[logcat.txt](https://github.com/wasdennnoch/AndroidN-ify/files/644543/logcat.txt)
[xposed_error_20161211_164240.log.txt](https://github.com/wasdennnoch/AndroidN-ify/files/644545/xposed_error_20161211_164240.log.txt)

**Comment by PradyunS** [comment]
Yes that happened to me too just turn off 'fix huge space under headerc

**Comment by LosTigeros** [comment]
Do how can I fix that huge space:?
https://drive.google.com/file/d/0B5IKSY3sCuE9bGVDZHR6bTlDaW8/view?usp=drivesdk
